charts/icm-web/values.yaml

```diff
@@ -23,7 +23,24 @@ webadapter:
     pullPolicy: IfNotPresent
   # disable HTTP/2 protocol support if required
   disableHTTP2: false
+
+  # Deprecated: The customHttpdConfig option has been replaced by configMapMounts.
+  # Please use configMapMounts to manually define the required ConfigMaps.
   customHttpdConfig: false
+
+  # List of ConfigMaps to be mounted as volumes
+  configMapMounts: []
+#    # the key will be part of the volume name (max 15 chars)
```

Comment:
Is this constraint enforced somehow? Why 15 chars?

Reply:
Issue